src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  return 'hello serhii';
```

Comment:
The line `return 'hello serhii';` is not part of the task requirements and should be removed as it will cause the function to exit early without performing the necessary calculations.